chrome-webstore-checklist.md

```diff
@@ -0,0 +1,19 @@
+# Chrome Web Store publication checklist
+
+## Permission justification
+
+To access most extension APIs and features, you must declare permissions in your extension's manifest. 
+Some permissions trigger warnings that users must allow to continue using the extension.
+
+Requested permissions:
+1. **activeTab**
+   
+    justification: to allow access to the currently active tab when the user invokes the extension. 
+2. **storage**:
+
+    justification: to be able to store user settings across sessions and browsers.
+3. **host_permissions**
```

Comment:
Shouldn't this then also be mentioned in the manifest.json?

Reply:
no, this needs to be entered when publishing on the Chrome Webstore.
There will be a form where all this is asked.

Reply:
You can only publish to the web store via an application which Google needs to approve. During the application process, Google will ask this info.